configs/components/echam/echam.datasets.yaml

```
@@ -133,3 +133,12 @@ forcing_sources:
                          to: 2014
                  "${input_dir}/MAC-SP_${scenario}.nc":
                          from: 2015
+
+        # carbon emissions for echam6:with_lco2_emis
```

Mini correction for formatting:
```suggestion
        # carbon emissions for echam6: with_lco2_emis
```

Still missing, which I still need to set up is a check to ensure that the DKRZ and AWI pool are identical. I did this once by hand, but automating it would be useful.